Source/DiabloUI/mainmenu.cpp

```diff
@@ -21,12 +21,14 @@ void UiMainMenuSelect(int value)
 
 void MainmenuEsc()
 {
+#ifndef NOEXIT

```

Comment:
Woudn't it make more sens to not add this to the menu rather then making it a no-op?